CHANGELOG.md

```diff
@@ -32,6 +32,8 @@
 - Add support for PixelSnap (via @dnicolson)
 - Add support for Cider (via @renews)
 - Add support for Typora (via @allenlsy)
+- Add default path for `XDG_CONFIG_HOME`. (via @HlubyLuk)
```

Comment:
Sorry, can you move this up to the WIP section?

Reply:
Ok, when arrive to work, I move to correct section.